lib/vdc/BOVCollection.cpp

```diff
@@ -181,6 +197,20 @@ int BOVCollection::_parseHeader(std::ifstream &header)
 
 int BOVCollection::_validateParsedValues()
 {
+    // Validate that the data file exists.
+    // If not given an absolute path, construct one
+    // from the header file's containing directory.
+    char  actualPath[PATH_MAX + 1];
```

Comment:
Is the function `realpath()` available on Windows?

Reply:
Nope.  Switched to use Wasp::FileUtils on current branch.